### PR TITLE
Add Krause House DAO

### PR DESCRIPTION
**Pull request by cryptosabago**

### CODE REVIEW

File: protocols/krausehouse/index.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "cname": "krausehouse",
+  "description": "Krause House is a DAO empowering fans to purchase and operate an NBA team.",
+  "path": "krausehouse",
+  "type": "snapshot",
+  "isEnabled": true,
+  "treasuryAddress": ["0x314e65DE454D78D4F685545824448371FeF4aac3"]
```
Comment (cryptosabago) [inline protocols/krausehouse/index.json:7]: not very confident about this one. waiting to hear back from kraus haus.

Comment (Rashmi-278) [reply protocols/krausehouse/index.json:7]: No worries :) Let's wait until we get a confirmation

**Review by Rashmi-278 — COMMENTED** [review]
In progress of confirming treasury address

File: protocols/krausehouse/index.json (hunk 1 above)
Comment (cryptosabago) [reply protocols/krausehouse/index.json]: Updated with the one that Tangfeng shared.

**Review by Rashmi-278 — APPROVED** [review]

**Pull request merged**